api-gateway/src/test/java/com/petclinic/bffapigateway/presentationlayer/ApiGatewayControllerTest.java

```diff
@@ -148,7 +160,30 @@ void createUser(){
         assertEquals(user.getUsername(), "Johnny123");
         assertEquals(user.getPassword(), "password");
         assertEquals(user.getEmail(), "email@email.com");
+    }
 
+    @Test
+    void updateUser(){
```

Comment:
Would probably need to change the naming convention of the test.

```suggestion
    void ShouldUpdateUserInformation(){
```